GetF!t/Modules/Exercises/Success/SuccessView.swift

```diff
@@ -31,10 +31,10 @@ struct SuccessView: View {
             }
 
             VStack {
-                HeaderView(selectedTab: .constant(0), titleText: LocalizedStringProvider.SuccesPage.success)
+                HeaderView(titleText: LocalizedStringProvider.SuccesPage.success)
                 Spacer()
                 RaisedButton(buttonText: LocalizedStringProvider.Button.continuE) {
-                    selectedTab = 9
+                    selectedTabManager.selectedTab = 9
```

Comment:
Tu priamo nastavuješ selected tab. Sprav na to nejakú funkciu, napr goToWelcomeTab alebo niečo take.  V selectedTabManager si nastav na selectedTab private set.